### DISCUSSION TITLE
Wert für #input.value setzen?

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#2445] [Q&A]
Ich würde gerne den Wert von einem input setzen können.
mit `getElementById(#input.id)._value` oder `getElementById(#input.id).value` hab ich es nicht hinbekommen.
So muss man die AnzeigeVarianten.Werte bei vorhandenen States in ioBroker nochmal setzen, obwohl ich in der Inputzeile anzeigen lassen kann.
Komme da nicht weiter, scheint intern abgefangen zu werden?

https://mcuiobroker.gitbook.io/jarvis-infos/jarvis-v3/besonderheiten-v3/scripts-3.1.x-pro/dropdown/getdisplayvaluesfromiobroker#scripts-getdisplayvaluesfromiobroker

**Comment by Zefau** [comment]
Glaube die Inputs haben keine `id`, oder? Also `getElementById` ginge dann nicht.

**Comment by mcuiobroker** [comment]
Ok, ich werde es ändern. Danke.
Grundsätzlich wäre es sinnvoll, wenn Du im Standard schon diese Option drin hättest, wenn `states` im Objekt zum Anzeige-DP vorhanden sind, diese zu laden?

**Reply by Zefau** [reply]
Verstehe noch nicht ganz, was du brauchst bzw. was du erzielen willst. Was genau fehlt dir und wo soll ich was bereitstellen?

**Comment by mcuiobroker** [comment]
z.B. Bei WLED gibt es für Funktion eine festgelegte Auswahl, auch schon in ioBroker.
Diese wird in den `states`   object.common.states gesetzt.
Man möchte diese gleiche Auswahl natürlich auch in jarvis haben.
Damit man nun nicht erst in ioBroker die states kopieren muss, sollte es von jarvis aus die Möglichkeit geben, dass man per Button oder automatisch diese `states` in die Anzeigevariante übernimmt.

![image](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/1f532dc5-6583-4a9f-b4af-358838a6f758)
![image](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/dc7b7362-c485-4d34-b0e7-c8b295170992)

Ziel automatisch hier drin
![image](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/af5faa5e-da1d-4194-8374-4a0757eea9c2)



id ist vorhanden
![image](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/a9e52aa9-6994-4a50-8c1f-f89f1f3ecfb5)
![jarvis v320-beta4 Input setzen](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/164bc28f-5959-476e-9102-8839e79a1ca8)
